settings.gradle.kts

```diff
@@ -319,13 +319,13 @@ include(":instrumentation:jedis:jedis-1.4:javaagent")
 include(":instrumentation:jedis:jedis-3.0:javaagent")
 include(":instrumentation:jedis:jedis-4.0:javaagent")
 include(":instrumentation:jedis:jedis-common:javaagent")
-include(":instrumentation:jetty:jetty-8.0:javaagent")
-include(":instrumentation:jetty:jetty-11.0:javaagent")
-include(":instrumentation:jetty:jetty-12.0:javaagent")
-include(":instrumentation:jetty:jetty-common:javaagent")
 include(":instrumentation:jetty-httpclient:jetty-httpclient-9.2:javaagent")
 include(":instrumentation:jetty-httpclient:jetty-httpclient-9.2:library")
 include(":instrumentation:jetty-httpclient:jetty-httpclient-9.2:testing")
+include(":instrumentation:jetty:jetty-11.0:javaagent")
+include(":instrumentation:jetty:jetty-12.0:javaagent")
+include(":instrumentation:jetty:jetty-8.0:javaagent")
```

Comment:
I think it would make sense to sort numbers in natural oder so that `8` comes before `11`. Intellij also orders the modules that way.

Reply:
I also prefer the modules ordered by version (earliest to latest, not ascii)

Reply:
> Using another rule may make the list become disordered again in the future.

The unfortunate reality is that if you don't enforce something, like for example we have spotless enforcing code style, then eventually things will get slightly disordered over time. I think this is not a big deal, as long as we have this mostly in reasonable order it is fine. Here we shouldn't fixate on what is the correct oder for the machine but what we expect as humans. That is why I prefer version numbers ascending from top to bottom instead of having them in lexicographical order.

Reply:
> The unfortunate reality is that if you don't enforce something

just run a grep/awk script to enforce the order?

Reply:
> > Using another rule may make the list become disordered again in the future.
> 
> The unfortunate reality is that if you don't enforce something, like for example we have spotless enforcing code style, then eventually things will get slightly disordered over time. I think this is not a big deal, as long as we have this mostly in reasonable order it is fine. Here we shouldn't fixate on what is the correct oder for the machine but what we expect as humans. That is why I prefer version numbers ascending from top to bottom instead of having them in lexicographical order.

Ok, I adjusted them.